docs/scripts/check_imports.py

```diff
@@ -0,0 +1,129 @@
+import importlib
```

Comment:
Could you add a module level description explaining what this code is designed to do?